miteD/service/client.py

```diff
@@ -1,17 +1,19 @@
 import logging
 from json import loads, dumps
+from nats.aio.errors import ErrTimeout
 
 from miteD.service.errors import MiteDRPCError
+from miteD.service.utils import format_version_str
 
 
 class RemoteService:
     def __init__(self, name, version, nc):
         self._logger = logging.getLogger('mited.RemoteService')
         self._nc = nc
         self._name = name
-        self._version = version.replace('.', '_')
+        self._version = format_version_str(version)
         self._proxy_cache = {}
-        self._prefix = '{}.{}'.format(name, version.replace('.', '_'))
+        self._prefix = 'rpc.service.{}.{}'.format(name, self._version)
```

Comment:
does this change affect any current `middlewares` that subscribe to an rpc service? If not then cool...

Reply:
Middleware / service layers need to be rebuilt and redeployed after that change.
Otherwise middleware will not be able to reach the service as the subject path would be out of sync

Reply:
The 'version' here is with regard to RPC method, not miteD release.
See miteD/examples/rpc_service.py - the service there defines 1.0 and 1.1 versions of 'ping' method.
By defining the version in notification_handler decorator you can make it listen to 1.0, 1.1 or either